### ISSUE TITLE
Warn ssh xyz ; abc will only half-execute

### ISSUE THREAD

**Issue by stdedos** [koalaman/shellcheck#1503]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
ssh -n "${i}" sudo apt-get update ; sudo apt-get install openssl=1.1.1b-0~ubuntu16.04.6+ppa.carsten+1 -y
```

#### Here's what shellcheck currently says:

_Nothing_

#### Here's what I wanted or expected to see:

`SC1234: Are you sure you meant to execute only one remote command? (';' stops the command)`

Note that e.g. `&&, ||, |` might be ... less-probable candidates for this linting, since, you might want to daisy-chain local commands from remote "status", or do "heavy-duty" processing on a microserver's output.

Not a "heavy" request, but more like something to discuss about.
